lib/Maintenance/Tasks/LogArchiveTask.php

```diff
@@ -77,7 +87,32 @@ public function execute()
                     ) ENGINE = ARCHIVE ROW_FORMAT = DEFAULT;");
 
             $db->query('INSERT INTO '.$tablename.' '.sprintf($sql, '*'));
-            $db->query('DELETE FROM '.ApplicationLoggerDb::TABLE_NAME.' WHERE `timestamp` < DATE_SUB(FROM_UNIXTIME('.$timestamp.'), INTERVAL '.$archive_treshold.' DAY);');
+            $db->query('DELETE FROM '.ApplicationLoggerDb::TABLE_NAME.' WHERE `timestamp` < DATE_SUB(FROM_UNIXTIME('.$timestamp.'), INTERVAL '.$archive_threshold.' DAY);');
+        }
+
+        $lockId = self::class;
+        if (!Lock::isLocked($lockId, 86400) && date('H') <= 4) {
```

Comment:
This hard-coded time window is not something I like. For example our QA testing instances on AWS are all on a schedule where they are shut down every night to save money. This would effectively never run on those instances. Can you make it configurable?

Reply:
This is what @brusch suggested in https://github.com/pimcore/pimcore/pull/7066#issuecomment-689345674 - for sure we can make that configurable. Shutting down your servers means that current implementation of `CacheCleanupTask` does not work for you either. @brusch what do you think?

Reply:
I'd keep it as it is for now, as we use the same approach also in different places. We can then create a follow up ticket where we can think about how to make that configurable or forceable.